modules/c++/scene/include/scene/ProjectionModel.h

```diff
@@ -95,6 +95,18 @@ class ProjectionModel
      *  R/Rdot Contour Ground Plane Intersection
      *
      */
+    static Vector3
```

Comment:
I'm not necessarily opposed to having a static method like this, but what's the use case where you want to just get the contour but don't want to construct a ProjectionModel?